pages.ja/windows/clip.md

```diff
@@ -0,0 +1,20 @@
+# clip
+
+> 入力コンテンツをWindowsクリップボードにコピーします
+> 詳しくはこちら: <https://docs.microsoft.com/windows-server/administration/windows-commands/clip>.
+
+- コマンドライン出力をWindowsクリップボードにパイプします:
+
+`{{dir}} | clip`
+
+- ファイルの内容をWindowsクリップボードにコピーします:
+
+`clip < {{path/to/file.ext}}`
```

Comment:
Do you think this should not be translated?

Reply:
I wanted to translate it to "file path" in Japanese, but I'm not sure how to translate the "ext" (file extension) part, and if they understand "file path" as in including the file extension. (I'm not very familiar with software engineer technical terms yet, frankly). Maybe a native Japanese can help, but that's the best I can do so far.

Reply:
ok. don't worry.

Reply:
Is it alright to leave it that way? If yes let's resolve this conversation :)